dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/patch/operation/ItemDiscoverableReplaceOperation.java

```diff
@@ -41,6 +58,32 @@ public R perform(Context context, R object, Operation operation) {
                 throw new UnprocessableEntityException("A template item cannot be discoverable.");
             }
             item.setDiscoverable(discoverable);
+
+            // Add suitable provenance
+            EPerson e = context.getCurrentUser();
+            String timestamp = DCDate.getCurrent().toString();
+            // Build some provenance data while we're at it.
```

Comment:
at what?

Reply:
this comment is from original dspace provenance comments but I modified it.

Reply:
better said, I took it from the comments when creating provenance dspace